Ssmpu/Memory_Protection_Unit.adoc

```diff
@@ -37,9 +37,12 @@ image::RV32_MPU_address_register_format.png[title="MPU address register format,
 image::RV64_MPU_address_register_format.png[title="MPU address register format, RV64"]
 
 The layout of MPU configuration registers is the same as PMP configuration registers, as is shown in Figure 6.
-The X/W/R field controls instruction execution, write and read permissions.
+The X/W/R bitsd controls instruction execution, write and read permissions.
+
+The S bit marks a rule as *S-mode-only* when set and *U-mode-only* when unset.
+The reserved encoding of ``mpucfg.RW=01``, and the encoding ``mpucfg.SRWX=1111``, now encode a Shared-Region.
```

Comment:
those weren't reserved before, mpucfg registers are defined fom scratch on this spec

Reply:
we also need to define what an entry/rule is (or maybe just say it's similar to ePMP)